### PR TITLE
Oxedions/move addons

### PR DESCRIPTION
**Pull request by oxedions**
Remove addons -> community
Keep some addons into core
Delete nic

### CODE REVIEW

File: resources/packaging/bluebanquise.spec [hunk 1]
```diff
@@ -103,26 +86,11 @@ cp -a tools/bluebanquise-playbook %{buildroot}%{_sbindir}/
...
-  print("Requires: " .. name .. " == " .. version .. "\n")
-  print("%description addons-" .. role .. "\n")
-  print("%files addons-" .. role .. " -f rolesfiles.addons." .. role .. "\n")
-end}
-
-
 %changelog
+* Thu Feb 25 2021 Benoit Leveugle <benoit.leveugle@gmail.com>
 * Sun Feb  2 2020 Bruno Travouillon <devel@travouillon.fr>
 * Tue Apr 14 2020 strus38 <indigoping4cgmi@gmail.com>
```
Comment (btravouillon) [inline resources/packaging/bluebanquise.spec:95]: The changelog is unordered ?!? Your change is correct, but last line is not. Could you use this PR to reorder the changelog?

**Review by btravouillon — CHANGES_REQUESTED** [review]

File: resources/examples/simple_cluster/playbooks/computes.yml [hunk 2]
```diff
@@ -9,8 +9,8 @@
       tags: set_hostname
     - role: repositories_client
       tags: repositories_client
-    - role: nic
-      tags: nic
+    - role: nic_nmcli
+      tags: nic_nmcli
```
[the remark this answers is not in the document]
Comment (btravouillon) [reply resources/examples/simple_cluster/playbooks/computes.yml:13]: On second thought I believe we should keep legacy `nic` until next major release (2.0), with a deprecation notice.
The change of the role may have an impact on users who already configured their system and rely on the role `nic` to configure their network interfaces.

File: .github/TEST_MATRIX.md [hunk 3]
```diff
@@ -27,7 +27,5 @@ Molecule unit tests
...
 |![advanced_core/advanced_dns_server unit testing](https://github.com/bluebanquise/bluebanquise/workflows/advanced_core/advanced_dns_server%20unit%20testing/badge.svg)                   | x | x |   |
-|![addons/clustershell unit testing](https://github.com/bluebanquise/bluebanquise/workflows/addons/clustershell%20unit%20testing/badge.svg)           | x | x | x |
-|![addons/grafana unit testing](https://github.com/bluebanquise/bluebanquise/workflows/addons/grafana%20unit%20testing/badge.svg)                     | x | x |   |
-|![addons/root_password unit testing](https://github.com/bluebanquise/bluebanquise/workflows/addons/root_password%20unit%20testing/badge.svg)         | x | x | x |
-|![addons/users_basic unit testing](https://github.com/bluebanquise/bluebanquise/workflows/addons/users_basic%20unit%20testing/badge.svg)             | x | x | x |
+|![addons/clustershell unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/clustershell%20unit%20testing/badge.svg)             | x | x | x |
```
[the remark this answers is not in the document]
Comment (oxedions) [reply .github/TEST_MATRIX.md:30]: I updated the link but not the string on the left. Corrected.

File: roles/core/nic_nmcli/vars/main.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
+---
+nic_nmcli_role_version: 1.3.0
```
[the remark this answers is not in the document]
Comment (oxedions) [reply roles/core/nic_nmcli/vars/main.yml:2]: Change was made on my local clone, I forgot to remove it. Reverted to previous one, will do the new one in a separate PR.

File: resources/packaging/bluebanquise.spec [hunk 5]
```diff
@@ -33,54 +29,43 @@ Requires:       python-netaddr
...
-hash_behaviour and groups.
+The stack is using Ansible, and relies heavily on variables precedence and groups.
 
 
 %prep
 %autosetup
 
 # Delete CICD files
-find roles/{core,advanced-core,addons} -maxdepth 2 \( -name 'molecule' -o -name '.ansible-lint' -o -name '.yamllint' \) -print0 \
+find roles/core -maxdepth 2 \( -name 'molecule' -o -name '.ansible-lint' -o -name '.yamllint' \) -print0 \
```
[the remark this answers is not in the document]
Comment (oxedions) [reply resources/packaging/bluebanquise.spec]: Done. I kept the - in addition to the _ in the ansible.cfg for now.

File: .github/TEST_MATRIX.md [hunk 6]
```diff
@@ -8,6 +8,7 @@ Molecule unit tests
 |                                                                                                                           | CentOS 8 | CentOS 7 | OpenSUSE 15.1 |
 |-----------------------------------------------------------------------------------------------------------------------------------------------------|:-:|:-:|:-:|
 |![core/bluebanquise unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/bluebanquise%20unit%20testing/badge.svg)               | x | x |   |
+|![core/clustershell unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/clustershell%20unit%20testing/badge.svg)             | x | x | x |
```
Comment (btravouillon) [inline .github/TEST_MATRIX.md:11]:
```suggestion
|![core/clustershell unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/clustershell%20unit%20testing/badge.svg)              | x | x | x |
```

File: .github/TEST_MATRIX.md [hunk 7]
```diff
@@ -21,11 +22,10 @@ Molecule unit tests
 |![core/pxe_stack unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/pxe_stack%20unit%20testing/badge.svg)                     | x | x |   |
 |![core/repositories_client unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/repositories_client%20unit%20testing/badge.svg) | x | x |   |
 |![core/repositories_server unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/repositories_server%20unit%20testing/badge.svg) | x | x |   |
+|![core/root_password unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/root_password%20unit%20testing/badge.svg)           | x | x | x |
```
Comment (btravouillon) [inline .github/TEST_MATRIX.md:25]:
```suggestion
|![core/root_password unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/root_password%20unit%20testing/badge.svg)            | x | x | x |
```

File: .github/TEST_MATRIX.md (hunk 6 above)
Comment (btravouillon) [reply .github/TEST_MATRIX.md:11]: To align columns in raw file.

**Comment by oxedions** [comment]
Seems ready. After that one, I will PR the new nmcli version and the RHEL likes.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**